pkg/helm/schema/helm.json

```diff
@@ -0,0 +1,234 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://schema.porter.sh/mixins/helm.json",
+  "definitions": {
+    "installStep": {
+      "type": "object",
+      "properties": {
+        "helm": {
+          "type": "object",
+          "properties": {
+            "description": {
+              "$ref": "http://schema.porter.sh/mixins/helm.json#/definitions/stepDescription"
```

Comment:
Right now it's important that mixins that use `$ref` use the fully qualified ref. Later, we can add an issue for porter to expand the ref automatically when it imports the mixin schema.